content/en/blog/_posts/2022-10-03-add-userns-alpha/index.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: blog
+title: User namespace alpha support
+date: 2022-10-03
+slug: userns-alpha
+canonicalUrl: https://gateway-api.sigs.k8s.io/blog/2022/user-namespace-alpha-support/
+---
+
+**Authors:** Rodrigo Campos (Microsoft), Giuseppe Scrivano (Red Hat)
+
+Kubernetes v1.25 introduces the support for user namespaces.
+
+This is a major improvement for running secure workloads in
+Kubernetes.  Each pod will have access only to a limited subset of the
+available UIDs and GIDs on the system, thus adding a new security
+layer to protect from other pods running on the same system.
+
+## How does it work?
+A process running on Linux can use up to 4294967296 different UIDs and
+GIDs.
+
+User namespaces is a Linux feature that allows mapping a set of users
+in the container to different users in the host, thus restricting what
+IDs a process can effectively use.
+Furthermore, the capabilities granted in a new user namespace do not
+apply in the host initial namespaces.
+
+## Why is it important?
+There are mainly two reasons why user namespaces are important:
+
+- improve security since they restrict the IDs a pod can use, so each
+pod can run in its own separate environment with unique IDs.
+
+- enable running workloads as root in a safer manner.
+
+In a user namespace we can map the root user inside the pod to a
+non-zero ID outside the container, containers believe in running as
+root while they are a regular unprivileged ID from the host point of
+view.
+
+The process can keep capabilities that are usually restricted to
+privileged pods and do it in a safe way since the capabilities granted
+in a new user namespace do not apply in the host initial namespaces.
+
+## How to enable user namespaces?
```

Comment:
If the next heading is “## How do I get involved?” then you could reword this too:
```suggestion
## How do I enable user namespaces?
```